pkg/virtctl/vm/vm.go

```diff
@@ -389,9 +434,93 @@ func getDryRunOption(dryRun bool) []string {
 	return nil
 }
 
-func (o *Command) Run(args []string) error {
+func expandArgs() cobra.PositionalArgs {
+	return func(cmd *cobra.Command, args []string) error {
+		if (cmd.Flags().Changed(filePathArg) || cmd.Flags().Changed(filePathArgShort)) && len(args) > 0 {
+			return fmt.Errorf("error file path and Virtual Machine name can not be both defined")
+		}
+
+		if !(cmd.Flags().Changed(filePathArg) || cmd.Flags().Changed(filePathArgShort)) && len(args) == 0 {
```

Comment:
Shouldn't we simply verify if `filePath` is empty, and if yes throw an error?

Reply:
You are correct, it is enough to just check `filePath`. I will change it also for the comment above.